ConnectionManager.java

```
@@ -0,0 +1,3 @@
+import java.sql.*;
+public class ConnectionManager{
+    public static Connection getConnection(){try{Connection con=DriverManager.getConnection("jdbc:mysql://localhost:3306/test","root","root");return con;}catch(Exception e){}return null;}}
```

Avoid using static methods for obtaining database connections. Consider using a connection pool instead. Also, the method should throw specific exceptions rather than catching a generic Exception and swallowing it.